public/js/lobby.js

```diff
@@ -1,15 +1,28 @@
 let data = {};
 
-const init = function() {
+function init() {
+    const room = createRoom();
     document.getElementById('button-send').addEventListener('click', send);
-}
+};
+
+function createRoom() {
+    let result = '';
+    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+    const charactersLength = 6;
+    for (let i = 0; i < length; i++) {
+        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+    }
+    return result;
+};
 
-const send = function(){
+function send() {
     let numImposters = document.getElementById('numImposters');
     let roundtime = document.getElementById('roundtime');
+    data['room'] = room;
     data['numImposters'] = parseInt(numImposters.value);
     data['roundtime'] = parseInt(roundtime.value);
```

Comment:
You could probably just use dot notation here since you already explicitly know which properties to modify.